qabelbox/src/main/java/de/qabel/desktop/repository/sqlite/hydrator/SimpleContactHydrator.kt

```diff
@@ -0,0 +1,45 @@
+package de.qabel.desktop.repository.sqlite.hydrator
+
+import de.qabel.core.config.Contact
+import de.qabel.core.crypto.QblECPublicKey
+import de.qabel.core.drop.DropURL
+import de.qabel.desktop.repository.EntityManager
+import de.qabel.desktop.repository.sqlite.schemas.ContactDB
+import org.spongycastle.util.encoders.Hex
+import java.sql.ResultSet
+import java.util.*
+
+class SimpleContactHydrator(private val entityManager: EntityManager) : AbstractHydrator<Contact>() {
+
+    override fun hydrateOne(resultSet: ResultSet?): Contact? {
+        var column = 1
+        val id = resultSet!!.getInt(column++)
```

Comment:
The resultset is not nullable. 
```@return the {@code ResultSet} generated by the query, never {@code null}.```
I think its just a kotlin<->java thing, so there are no null values. It can produce exceptions if the resultset is not checked for entries or the query is not matching the hydrator.